### PR TITLE
Add meta tags and description

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

**Comment by ZJvandeWeg** [comment]
It's best if I don't review anything in this repo, reassigning.

File: docs/install/local/stacks.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 ---
 navTitle: Local Stacks
+meta:
+   description: Configure Node-RED instances with custom memory limits and versions using Stacks in FlowFuse for local deployments.
+   tags:
+      - flowfuse
+      - nodered
+      - local developement
```
Comment (knolleary) [inline docs/install/local/stacks.md:8]:
```suggestion
      - local development
```

File: docs/contribute/introduction.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
 ---
 navTitle: Introduction
 navOrder: 1
+meta:
+    description: This guide provides detailed instructions on how to contribute to the FlowFuse project, including setup, development, and testing processes.
+    tags:
+       - flowfuse
+       - developement
```
Comment (knolleary) [inline docs/contribute/introduction.md:8]:
```suggestion
       - development
```

**Review by knolleary — APPROVED** [review]
Approved pending a couple typo fixes

**Pull request merged**